generators/client/templates/blazor/test/Project.Client.Test/AlertErrorTest.cs.ejs

```diff
@@ -49,7 +49,7 @@ public class AlertErrorTest : TestContext
             .Replace(ServiceDescriptor.Transient<IComponentActivator, ComponentActivator>())
             .AddBootstrapProviders()
             .AddFontAwesomeIcons();
-
+        JSInterop.Mode = JSRuntimeMode.Loose;
```

Comment:
Why is this needed?

Reply:
Hi @ivanmonteiro  
Without this setting, the test is showing like this photo 
![image](https://user-images.githubusercontent.com/4284170/229376100-78273f13-7afc-4e88-9bc8-725fe8f3fc2e.png)

I tried to add "SetupModule("./_content/Blazorise/utilities.js?v=1.2.2.0")"
But it's so annoying with this error after
" Bunit.JSRuntimeUnhandledInvocationException : bUnit's JSInterop has not been configured to handle the call:

    InvokeVoidAsync("log", "%cThank you for using the free version of the Blazorise component library! We're happy to offer it to you for personal use. If you'd like to remove this message, consider purchasing a commercial license from https://blazorise.com/commercial. We appreciate your support!", System.String[])

Configure bUnit's JSInterop to handle the call with following:

    SetupVoid("log", "%cThank you for using the free version of the Blazorise component library! We're happy to offer it to you for personal use. If you'd like to remove this message, consider purchasing a commercial license from https://blazorise.com/commercial. We appreciate your support!", System.String[])
or the following, to match any arguments:
    SetupVoid("log", _ => true)

The setup methods are available on an instance of the BunitJSInterop or
BunitJSModuleInterop type. The standard BunitJSInterop is available
through the TestContext.JSInterop property, and a BunitJSModuleInterop
instance is returned from calling SetupModule on a BunitJSInterop instance.
"

![image](https://user-images.githubusercontent.com/4284170/229376148-06e31206-b495-41de-9b1a-bd701a7dbf09.png)

Maybe we can find out another way after update or if you have any other idea for this.